### PR TITLE
test: add a test to check various children set to false

### PR DESCRIPTION
**Pull request by BrandonOCasey**
* Added a test that should catch assertions that happen when a child is turned off on the player.
* About to add a fix for the current assertion on liveTracker/seekToLive.

### CODE REVIEW

File: test/unit/player.test.js [hunk 1]
```diff
@@ -1991,3 +1991,19 @@ QUnit.test('source options are retained', function(assert) {
...
+
+QUnit.test('Player is fine with most children off', function(assert) {
+  const defaultChildren = Player.prototype.options_.children;
+
+  defaultChildren.forEach((childName) => {
+    const options = {};
+
+    options[childName] = false;
+
+    const player = TestHelpers.makePlayer(options);
```
[the remark this answers is not in the document]
Comment (BrandonOCasey) [reply test/unit/player.test.js]: I think we would want the opposite, which does not exist. After initially wrapping these in a `try/catch` block. I think we will get better reporting just letting the assertion happen.

[the remark this answers is not in the document]
Comment (gkatsev) [reply test/unit/player.test.js]: ah, right, we want a "notThrows" which doesn't exist. This makes sense then.

**Comment by gkatsev** [comment]
Can the `sed` command be run to fix the https->http issue in package-lock? Otherwise, LGTM

**Comment by BrandonOCasey** [comment]
`sed` command done

**Pull request merged**